### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheAutomaTom**
Run spellcheck.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,17 @@
 # unplugin-auto-expose
 <a href="https://www.buymeacoffee.com/kozack" target="_blank"><img src="https://cdn.buymeacoffee.com/buttons/v2/default-red.png" height="60" alt="Buy Me A Coffee"></a>
 
-Plugins for automatic `exposeInMainWorld` everething you exported from preload and easely importing exposed api in renderer.
+Plugins for automatic `exposeInMainWorld` everything you exported from preload and easily importing exposed api in renderer.
 
 ## Example
 ```ts
-// preload.ts
```
Comment (cawa-93) [inline README.md]: Comments with `//` is ok

File: README.md [hunk 2]
```diff
@@ -59,8 +59,10 @@ export default defineConfig({
 })
 ```
 ## TypeScript
-To configurate the TypeScript, add a path to your renderer `tsconfig.json`:
+To configure the TypeScript, add a path to your renderer 
+
 ```json
+/// tsconfig.json`:
```
Comment (cawa-93) [inline README.md]: Comments in json is not ok

**Review by cawa-93 — CHANGES_REQUESTED** [review]

**Comment by cawa-93** [comment]
Thanks for checking. And sorry for delay 😔

**Comment by TheAutomaTom** [comment]
Never a rush.  I hope you have more important things to do!

**Pull request merged**